src/Swashbuckle.AspNetCore.SwaggerGen/Annotations/SwaggerIgnoreAttribute.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+// ReSharper disable once CheckNamespace
+namespace Swashbuckle.AspNetCore.Annotations
+{
+    /// <summary>
+    /// Causes the annotated member to be ignored during schema generation.
+    /// Does not alter serialization behavior.
+    /// </summary>
+    /// <remarks>
+    /// Can be used in combination with <see cref="System.Text.Json.Serialization.JsonExtensionDataAttribute"/>
+    /// to capture and invalidate unsupported properties.
+    /// </remarks>
+    [AttributeUsage(AttributeTargets.Method | AttributeTargets.Parameter | AttributeTargets.Property)]
+    public class SwaggerIgnoreAttribute : Attribute { }
```

Comment:
We can always unseal it if a use case for extension ever appears.

```suggestion
    public sealed class SwaggerIgnoreAttribute : Attribute { }
```